src/components/Navibar.js

```diff
@@ -1,33 +1,38 @@
-import React from 'react'
-import {Container, Nav, Navbar} from 'react-bootstrap'
-import 'bootstrap/dist/css/bootstrap.min.css';
-import { Link } from 'react-router-dom';
-
-import '../styles/style.css'
+import React from "react";
+import { Container, Nav, Navbar } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Link } from "react-router-dom";
 
+import "../styles/style.css";
 
 export default function Navibar() {
```

Comment:
use construction like this: `
const Component = () => {.......}
export default Component
`

Use export for additional functionality or utils